Dockerfile

```diff
@@ -78,7 +78,7 @@ RUN pip install mcstatus
 ###################
 ### Healthcheck ###
 ###################
-HEALTHCHECK --interval=10s --timeout=5s \
+HEALTHCHECK --interval=10s --timeout=5s --start-period=120s \
```

Comment:
Aren't two minutes too much? 60s and a 30s interval sound more resonable. It wont fail fast but will have enough warm up time.

Reply:
I wouldn't increase the interval because it takes `n` failing health checks (default 3) for a container to transition to "unhealthy". With an interval of 10s a faulty container is restarted after 30sec instead of 90s (if interval is increased to 30s). 

My server takes about 57sec to start so I set the start-period here to twice that amount. Keep in mind that this doesn't affect the health checks after the first 120sec. Also I'd rather provide a reliably long start-period over risking false-positive "unhealthy" transitions because the start-period was chosen too conservatively.

Reply:
When I restarted my server earlier it logged the start up time at almost 64sec:
`[16:38:02 INFO]: Done (63.395s)! For help, type "help"`